src/main/java/net/amygdalum/tanteemmas/external/TimeProvider.java

```diff
@@ -2,11 +2,11 @@
 
 public class TimeProvider {
 
-	private long start;
```

Comment:
Why do we need these changes?

Reply:
Preparing the code for recheck-web. We want it complicated but deterministic...